app/controllers/katello/api/v2/repositories_controller.rb

```diff
@@ -90,6 +90,7 @@ def custom_index_relation(collection)
     param :description, String, :desc => N_("description of the repository")
     param :available_for, String, :desc => N_("interpret specified object to return only Repositories that can be associated with specified object.  Only 'content_view' & 'content_view_version' are supported."),
           :required => false
+    param :with_content, String, :desc => N_("only repositories having at least one of the specified content type ex: rpm , erratum")
```

Comment:
Do we have any precedent in the code that can autogenerate the available types for the api docs? I seem to recall something but not sure where

Reply:
Right..I thought about that but couldn't see a straightforward way to get all content_types..I'll think about this a little..